### PR TITLE
Remove scratch

### PR DESCRIPTION
**Pull request by hakonfam**
Get rid of not-required scratch area and use move-swap instead.

### CODE REVIEW

File: boot/zephyr/prj.conf [hunk 1]
```diff
@@ -45,3 +45,6 @@ CONFIG_LOG=y
 CONFIG_LOG_IMMEDIATE=y
 ### Ensure Zephyr logging changes don't use more resources
 CONFIG_LOG_DEFAULT_LEVEL=0
+
+# Save space by not using scratch partition, but smart swap instead.
+CONFIG_BOOT_SWAP_USING_MOVE=y
```
Comment (nvlsianpu) [inline boot/zephyr/prj.conf]: y will be the default value for NRF, `temphack` or remove?

**Comment by hakonfam** [comment]
Closing as this will be fixed upstream

**Comment by hakonfam** [comment]
Forgot what this PR was about, will remove the commit now in ustream.

**Comment by hakonfam** [comment]
IMO there is no need to wait for upmerge, as this has the desired effect in the existing code. The user can enable this option already, its just not the default.

**Comment by nvlsianpu** [comment]
OK. so lets wait for current upmerge

**Comment by nvlsianpu** [comment]
@hakonfam bump version in nrf manifest

**Pull request merged**